### PR TITLE
Create lyft.json

### PR DESCRIPTION
**Pull request by johandekeulenaer**

### CODE REVIEW

File: data/lyft.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "countryHQ": "US",
+  "gdprReadyStatus": "nonCompliant",
+  "privacyUrl": "https://www.lyft.com/privacy",
+  "dsarUrl": "",
+  "dpaUrl": "",
+  "subprocessorsUrl": "",
+  "dataCenters": [
+    "US"
+  ],
+  "hostingPartners": [
```
Comment (Gdewilde) [inline data/lyft.json]: we renamed this to hostingProviders

File: data/lyft.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "description": "Lyft is an on-demand transportation company",
+  "categories": [
+    "SaaS", 
+    "Transportaion"
+  ],
+  "website": "https://www.lyft.com",
+  "applicationUrl": "https://account.lyft.com/auth",    
+  "twitter": "lyft",
+  "countryHQ": "US",
+  "gdprReadyStatus": "nonCompliant",
```
Comment (Gdewilde) [inline data/lyft.json]: Link to an article to support this claim

**Review by Gdewilde — COMMENTED** [review]
Relevant link to add: https://www.theverge.com/2018/1/25/16934064/lyft-customer-data-abuse-allegations

File: data/lyft.json (hunk 1 above)
Comment (johandekeulenaer) [reply data/lyft.json]: Has been updated in the edit template

**Pull request merged**